src/components/AttachmentPicker/index.native.js

```diff
@@ -174,6 +103,80 @@ class AttachmentPicker extends Component {
         }
     }
 
+    /**
+     * Inform the users when they need to grant camera access and guide them to settings
+     */
+    showPermissionsAlert() {
+        Alert.alert(
+            this.props.translate('attachmentPicker.cameraPermissionRequired'),
+            this.props.translate('attachmentPicker.expensifyDoesntHaveAccessToCamera'),
+            [
+                {
+                    text: this.props.translate('common.cancel'),
+                    style: 'cancel',
+                },
+                {
+                    text: this.props.translate('common.settings'),
```

Comment:
No such key `common.settings` in languages/en: https://github.com/Expensify/Expensify.cash/pull/2718/files#diff-22037c1e4dbdea8803b450b3fce06e6c475762a31ecf84b34dc264ba0e443637R3

If it ever becomes the case that the user revoked camera permissions the app will seem to do nothing when they press on "Add Attachment" -> "Take Photo". 
While the app actually throws an error here 

https://user-images.githubusercontent.com/12156624/121061991-38cb4200-c7cd-11eb-8d38-6be3e5921773.mp4

The problem will be worse if the user selects "Deny & don't ask" as nothing would seem to happen after you press on "Take Photo"

- Expected: When the app lacks permissions to take photos for some reason it should show alert message explaining why the "Take Photo" button doesn't work 
- Actual: Pressing "Take Photo" does nothing